idiokit/dns/tests/test_dns_conf.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+import tempfile
+
+from .. import _conf
+
+
+class HostsFileTests(unittest.TestCase):
+    _missing_file = None
+    _empty_file = None
+    _hosts = None
```

Comment:
You don't really need to declare these (class) attributes before use. They do serve as documentation of sorts, but see my comments about setUp.